src/Libraries/Microsoft.Extensions.AI.Abstractions/Functions/AIFunctionMetadata.cs

```diff
@@ -100,6 +104,43 @@ public AIFunctionReturnParameterMetadata ReturnParameter
         init => _returnParameter = Throw.IfNull(value);
     }
 
+    /// <summary>Gets a JSON Schema describing the function and its input parameters.</summary>
+    /// <remarks>
+    /// <para>
+    /// When specified, declares a self-contained JSON schema document that describes the function and its input parameters.
+    /// A simple example of a JSON schema for a function that adds two numbers together is shown below:
+    /// </para>
+    /// <code>
+    /// {
+    ///   "title" : "addNumbers",
+    ///   "description": "A simple function that adds two numbers together.",
+    ///   "type": "object",
+    ///   "properties": {
+    ///     "a" : { "type": "number" },
+    ///     "b" : { "type": "number", "default": 1 }
+    ///   }, 
+    ///   "required" : ["a"]
```

Comment:
It's a bit inconsistent now that we have the return schema contained separately from the function schema. Is there any standard way with schemas for functions to represent a return value, such that we could augment the function schema with it rather than having it in the separate return instance? I realize none of the AI vendors care about it today.

Reply:
Not that I could find. Unless the AI vendors legislate a schema keyword for the return type into existence, this would need to be kept as a separate property for now. It don't believe it's going to be necessary though -- I think we might as well remove it.